packages/deployer/src/build/watcher.ts

```diff
@@ -15,6 +20,7 @@ export async function getInputOptions(entryFile: string, platform: 'node' | 'bro
     },
     platform,
     env,
+    sourcemapEnabled,
```

Comment:
Let's make this an object so we can pass more bundler options in the future
```suggestion
    {sourcemap: sourcemapEnabled},
```

Reply:
Agreed! Updated in latest commit.